app/components/markdown/markdown_link/markdown_link.js

```diff
@@ -55,7 +57,16 @@ export default class MarkdownLink extends PureComponent {
         const match = matchDeepLink(url, serverUrl, siteURL);
         if (match) {
             if (match.type === DeepLinkTypes.CHANNEL) {
-                this.props.actions.handleSelectChannelByName(match.channelName, match.teamName);
+                const response = await this.props.actions.handleSelectChannelByName(match.channelName, match.teamName);
+
+                if (response && response.error) {
+                    const linkFailedMessage = {
+                        id: t('permalink.error.access'),
```

Comment:
This id does not match, also changes need to be added in en.json

Reply:
It's there: https://github.com/mattermost/mattermost-mobile/blob/master/assets/base/i18n/en.json#L516